ape_etherscan/client.py

```diff
@@ -25,6 +27,11 @@ def get_etherscan_api_uri(network_name: str):
     )
 
 
+class SourceCodeResponse(BaseModel):
+    ABI: str = ""
+    ContractName: str = "unknown"
```

Comment:
Feel like pydantic might be too heavy for this small use case, was picturing something more like straight dataclasses

Plus the potential for conflicting pins on pydantic vs. core scare me